### DISCUSSION TITLE
fblog as an alternative to penna-dev

### DISCUSSION THREAD

**Discussion by agentgt** [hkupty/maple#73] [Ideas]
Hi @hkupty 

I have been using fblog for ad-hoc JSON logs that for whatever reason are not aggregated into a central system: https://github.com/brocode/fblog

As I promised I try to share things that I think could help or interest penna.

One thing about fblog is that you have to configure it to map the fields so there might be some sort of automation that could be done to generate say a proper bash script.

**Comment by hkupty** [comment]
That's really interesting! Thanks a lot! I'll play around with that and maybe add some automation for Penna as you said :)

**Comment by hkupty** [comment]
I've been thinking about it more lately, specially after trying fblog a bit and I think it makes sense to simplify on the logger side since there's fblog (and other tools) available. Reverting/retiring penna-dev is on the roadmap :)
